fury/ui/tests/test_helpers.py

```diff
@@ -47,3 +47,46 @@ def test_clip_overflow():
 
     npt.assert_raises(ValueError, clip_overflow,
                       text, rectangle.size[0], 'middle')
+
+
+def test_wrap_overflow():
+    text = ui.TextBlock2D(text="", position=(50, 50), color=(1, 0, 0))
+    rectangle = ui.Rectangle2D(position=(50, 50), size=(100, 50))
+
+    sm = window.ShowManager()
+    sm.scene.add(rectangle, text)
+
+    text.message = "Hello"
+    wrap_overflow(text, rectangle.size[0])
+    npt.assert_equal("Hello", text.message)
+
+    text.message = "Hello wassup"
+    wrap_overflow(text, rectangle.size[0])
+    npt.assert_equal("Hello wassu\np", text.message)
+
+    text.message = "A very very long message to clip text overflow"
+    wrap_overflow(text, rectangle.size[0])
+    npt.assert_equal("A very very\n long mess\nage to clip \ntext overflo\nw",
+                     text.message)
+
+    text.message = "A very very long message to clip text overflow"
+    wrap_overflow(text, 0)
+    npt.assert_equal(text.message, "")
+
+    wrap_overflow(text, -2*text.size[0])
+    npt.assert_equal(text.message, "")
+
+
+def test_check_overflow():
+    text = ui.TextBlock2D(text="", position=(50, 50), color=(1, 0, 0))
+    rectangle = ui.Rectangle2D(position=(50, 50), size=(100, 50))
+
+    sm = window.ShowManager()
+    sm.scene.add(rectangle, text)
+
+    text.message = "A very very long message to clip text overflow"
+
+    is_overflowing, _ = check_overflow(text, rectangle.size[0], '~')
+
+    npt.assert_equal(False, is_overflowing)
```

Comment:
This should return `True` not `False` as it indeed is overflowing.

Reply:
Test have been updated accordingly.